.github/workflows/continuous-integration.yml

```diff
@@ -65,21 +72,28 @@ jobs:
   linux-compat:
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
       matrix:
         image:
-          - aws-crt-swift-5-al2-x64
+          - ubuntu-x64
+          - al2-x64
     steps:
       - name: Checkout Sources
         uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '11'
       - name: Build and Test ${{ env.PACKAGE_NAME }}
         run: |
-          echo "${{ secrets.GITHUB_TOKEN }}" | docker login docker.pkg.github.com -u awslabs --password-stdin
-          export DOCKER_IMAGE=docker.pkg.github.com/awslabs/aws-crt-builder/${{ matrix.image }}:${{ env.BUILDER_VERSION }}
+          aws ecr get-login-password | docker login 123124136734.dkr.ecr.us-east-1.amazonaws.com -u AWS --password-stdin
```

Comment:
the crt builder made some major updates and since we had to update our version to use the latest swift images, we had to also update our run script. one of the changes they made is they no longer store their images in docker, and now use ECR. this change accounts for that.